imod_coupler/drivers/dfm_metamod/msw_wrapper.py

```diff
@@ -0,0 +1,213 @@
+from pathlib import Path
+from typing import Any, Optional, Union
+
+import numpy as np
+from numpy.typing import NDArray
+from xmipy import XmiWrapper
+
+
+class MswWrapper(XmiWrapper):
+    def __init__(
+        self,
+        lib_path: Union[str, Path],
+        lib_dependency: Union[str, Path, None] = None,
+        working_directory: Union[str, Path, None] = None,
+        timing: bool = False,
+    ):
+        super().__init__(lib_path, lib_dependency, working_directory, timing)
+
+    def initialise_surfacewater_component(self) -> None:
+
+        """
+        function to initialise the surface water component in metaswap.
+
+        Parameters
+        ----------
+        none
+
+        """
+        idtsw = 0
```

Comment:
this function creates 2 local variables, sets them to 0, and then it is finished? that's what it looks like anyway. then it could be removed!